endpoints/getting-started-grpc/README.md

```diff
@@ -44,11 +44,10 @@ $ protoc \
 
 1. In `api_config.yaml`, replace `MY_PROJECT_ID` and `SERVICE-ACCOUNT-ID` with your Project ID and your service account's email address respectively.
 
-1. Deploy your service's configuration to Endpoints. Take note of your service's config ID and name once the deployment completes.
+1. Deploy your service's configuration to Endpoints. 
 ```
 $ gcloud endpoints services deploy api.pb api_config.yaml
 ...
-Service Configuration [SERVICE_CONFIG_ID] uploaded for service [SERVICE_NAME]
```

Comment:
This was the only place where we define what SERVICE_NAME is. Without this, users may be confused about what SERVICE_NAME is.

I think we should leave this here, even if we don't end up using SERVICE_CONFIG_ID.